packages/console/src/SerialConsole/SerialConsole.js

```diff
@@ -143,6 +145,10 @@ SerialConsole.propTypes = {
   rows: PropTypes.number,
   cols: PropTypes.number,
 
+  /** Font for text rendered to xterm canvas */
+  fontFamily: PropTypes.string,
```

Comment:
Should we depend on the xterm default here?
There will be `Menlo, Monaco, Consolas, monospace` of size `12` used in Cockpit.

Would be great to do this styling via CSS but recent xterm seems to enforce this property.

Reply:
@patternfly/patternfly-react-ux thoughts?

Reply:
Perhaps this could be good to encourage better readability?

Reply:
Actually it's in a JS comment bellow for the `defaultProps`.
I just positioned this comment on GitHub incorrectly.